default/stringtables/en.txt

```diff
@@ -13415,6 +13430,12 @@ Evacuation System
 BLD_EVACUATION_DESC
 Remove [[metertype METER_POPULATION]] from this planet over several turns according to an [[encyclopedia EVACUATION_TITLE]]. If a suitable alternate planets are available, with the same species and sufficient extra capacity, Population will move there.
 
+BLD_ABANDON_OUTPOST
+Abandon System
```

Comment:
Remove / replace "System" in the name. Makes purpose unclear.

Reply:
Should I also remove "System" from the "Evacuation System" building? (I think the system part was introduced to gap the difference between a building and an activity)

Reply:
For consistency, probably yes.

To be clear, in the case of "Evacuation System", it's not ambiguous whether "System" refers to a star system vs. a means and process for doing something (ie. the evacuation).